packages/ozone/src/db/migrations/20250609T110704000Z-safelink.ts

```diff
@@ -0,0 +1,71 @@
+import { Kysely, sql } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .createTable('safelink_event')
+    .addColumn('id', 'bigserial', (col) => col.primaryKey())
+    .addColumn('eventType', 'varchar', (col) => col.notNull())
+    .addColumn('url', 'varchar', (col) => col.notNull())
+    .addColumn('pattern', 'varchar', (col) => col.notNull())
+    .addColumn('action', 'varchar', (col) => col.notNull())
+    .addColumn('reason', 'varchar', (col) => col.notNull())
+    .addColumn('createdBy', 'varchar', (col) => col.notNull())
+    .addColumn('createdAt', 'varchar', (col) =>
+      col.defaultTo(sql`now()`).notNull(),
+    )
+    .addColumn('comment', 'text')
+    .execute()
+
+  await db.schema
+    .createTable('safelink')
```

Comment:
Perhaps `safelink_rule` for consistency with the parlance used in the lexicon, etc?